packages/docs/docs/bots/bot-cron-job.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Great work, love the screenshots! 
Can you add a single small section on the [bots-basics page](https://github.com/medplum/medplum/blob/main/packages/docs/docs/bots/bot-basics.md#executing-a-bot), after  the [Execute Automatically](https://github.com/medplum/medplum/blob/main/packages/docs/docs/bots/bot-basics.md#executing-automatically-using-a-subscription) section? Can just be 1-2 sentences that link to this page.